portfolio/src/main/webapp/script.js

```diff
@@ -12,10 +12,18 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+console.log("reached here")
+
+function fetchfunction(){
+    fetch("/data").then(response => response.json()).then((json) => {
+        document.getElementById('testing').innerText = json;
+        console.log(json);
```

Comment:
this log is actually maybe okay to keep if you wanted to. It might be useful for a user to see the information returned from the server (and you don't need to hide it because this information is being added to the page anyway). However, it's also a good idea to remove it if you don't need it and if it doesn't provide any extra value.